model_prices_and_context_window.json

```diff
@@ -44,56 +59,81 @@
         "input_cost_per_token": 0.0000015,
         "output_cost_per_token": 0.000002
     },
-    "claude-instant-1": {
-        "max_tokens": 100000,
-        "input_cost_per_token": 0.00000163,
-        "output_cost_per_token": 0.00000551
-    },
-    "claude-instant-1.2": {
-        "max_tokens": 100000,
-        "input_cost_per_token": 0.00000163,
-        "output_cost_per_token": 0.00000551
-    },
-    "claude-2": {
-        "max_tokens": 100000,
-        "input_cost_per_token": 0.00001102,
-        "output_cost_per_token": 0.00003268
+    "text-ada-001": {
+        "max_tokens": 2049,
+        "input_cost_per_token": 0.0000004,
+        "output_cost_per_token": 0.0000004
     },
-    "text-bison-001": {
```

Comment:
can we not delete models

Reply:
@ishaan-jaff Look further down the file. They are included, just moved to align with the where they are defined in `__init__.py`